connector_acp_ftp/models/backend_acp.py

```diff
@@ -61,8 +61,7 @@ def action_confirm(self):
         if self.connection_type == "ftp":
             self._ftp_upload(None)
             return True
-        else:
-            return super().action_confirm()
+        return super().action_confirm()
```

Comment:
@dreispt If the FTP connection works, True will be returned and the state will not be updated.

```
        if self.connection_type == "ftp":
￼            self._ftp_upload(None)
￼            return super().action_confirm()
```

Reply:
Good point. But I think the solution is to remove the return statement inside the if block.